src/libraries/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Unix.cs

```diff
@@ -226,9 +226,10 @@ internal static string GetPipePath(string serverName, string pipeName)
             return s_pipePrefix + pipeName;
         }
 
+#pragma warning disable CA1822
         /// <summary>Throws an exception if the supplied handle does not represent a valid pipe.</summary>
         /// <param name="safePipeHandle">The handle to validate.</param>
-        internal void ValidateHandleIsPipe(SafePipeHandle safePipeHandle)
+        private protected void ValidateHandleIsPipe(SafePipeHandle safePipeHandle)
```

Comment:
Why did the visibility here change?  I'm ok with it, I just don't see the connection to the rest of the PR.

Reply:
I think it was to help to test static on the method. I will remove the change.